src/components/EditProfile/edit.css

```diff
@@ -0,0 +1,57 @@
+.edit-form {
```

Comment:
Not sure we need this file. AntD should be used for most styling.

Reply:
It was taking me a long time to come up to speed with ant-design so i styled everything with css. I'd be happy if u could walk me through changing it to ant design

Reply:
@shaunorpen